src/index.ts

```diff
@@ -61,4 +77,38 @@ export default class IntegrationTestEnvironment extends NodeEnvironment {
         this.global.ceramicClient = ceramicClient
         this.global.ipfs = ipfs
     }
+
+    private async cleanStateStoreIfPrivate() {
+        console.info(`checking on state store in environment ${process.env.NODE_ENV}`)
+        const bucket_name = config.jest.services.ceramic.s3StateStoreBucketName
+
+        if ( ! bucket_name ) {
+            console.info("Nothing to do")
+            return
+        }
+
+        const parts = bucket_name.split("/", 2)
+
+        if ( ! parts[0].includes('test')) {
```

Comment:
also does this do anything with the buckets anymore? it seems like we just remove all pinned streams

Reply:
it doesn't hit the buckets directly, the files in s3 are removed with the streams - is safer than direct s3 walk

Reply:
but, I am paranoid about making sure we don't do it unless we are in a test bucket, bc if we call this at the wrong time we could delete all the private node state store for instances.  in case one time someone points the integration tests at the wrong target I don't want to destroy some node's state store.